search.php

```diff
@@ -980,6 +980,7 @@
 			{
 				$folder .= "new";
 				$folder_label .= $lang->icon_new;
+				$thread['newpostlink'] = get_thread_link($post['thread_lastpost']);
```

Comment:
Isn't this link supposed to point to the last post?

Reply:
Fixed, thx!